games/modes/team.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Team mode
+ * Cassius - https://github.com/sirDonovan/Cassius
+ *
+ * This file contains code for a game mode (Team)
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+const name = 'Team';
+const id = Tools.toId(name);
+const data = {};
+
+data["Teams"] = Tools.data.teams;
+
+let TeamMode = function () {
+	this.name += ' ' + name;
+	this.id += id;
+	this.freeJoin = false;
+	this.playerList = [];
+	this.teamA = '';
+	this.teamB = '';
+	this.teamAplayers = [];
+	this.teamBplayers = [];
+	this.points = new Map();
+	this.maxPoints = 20;
+	this.onSignups = null;
+
+	this.onStart = () => {
+		if (this.playerCount < 2) {
+			this.say("This game mode requires at least two players to begin!");
+			this.end();
+			return;
+		}
+		let tracker = Object.keys(this.players), str = 0, teamNames = data["Teams"];
+		this.teamA = Object.keys(teamNames)[Math.floor(Math.random() * Object.keys(teamNames).length)];
+		this.teamB = teamNames[this.teamA];
+		let randomPlayer = Tools.shuffle(tracker);
+		for (let i = 0, len = tracker.length; i < len; i++) {
+			if (str % 2 === 0) {
+				this.players[randomPlayer[i]].team = this.teamA;
+				this.teamAplayers.push(this.players[randomPlayer[i]].name);
+				str++;
+			} else {
+				this.players[randomPlayer[i]].team = this.teamB;
+				this.teamBplayers.push(this.players[randomPlayer[i]].name);
+				str++;
+			}
+		}
+		this.say("Players (" + this.playerCount + ") **" + this.teamA + ":** " + this.teamAplayers.join(", ") + " | **" + this.teamB + ":** " + this.teamBplayers.join(", ") + ".");
+		this.nextRound();
+	};
+
+	this.onNextRound = () => {
+		this.setAnswers();
+		console.log(this.players);
+		let placeholder = true;
+		this.say(this.hint);
+		this.timeout = setTimeout(() => {
+			if (placeholder) {
```

Comment:
<code>placeholder</code> isn't modified anywhere after it's declared. This can just check <code>if (this.answers)</code> anyway.